bdpy/ml/learning.py

```diff
@@ -230,27 +236,41 @@ class ModelTraining(object):
         Verbosity level.
     '''
 
-    def __init__(self, model, X, Y):
+    def __init__(
+            self, model: SupportsFit, X: np.ndarray, Y: np.ndarray,
+            X_normalize: Optional[np.ndarray] = None,
+            Y_normalize: Optional[np.ndarray] = None,
+            X_sort: Optional[np.ndarray] = None,
+            Y_sort: Optional[np.ndarray] = None,
```

Comment:
`X_sort` and `Y_sort` should be like `{'index': <np.ndarray>}`. So, the appropriate type is `Optional[Dict[str, np.ndarray]]`